LongWarOfTheChosen/Src/LW_Overhaul/Classes/LW2_UIImageSelector.uc

```diff
@@ -0,0 +1,294 @@
+// this is mostly the same as UIColorSelector
+class LW2_UIImageSelector extends UIPanel;
+
+var float ITEM_PADDING;
+var float EDGE_PADDING;
+var float				ChipHeight; 
+var float				ScrollbarPadding;
+var UIScrollbar		Scrollbar;
+var UIMask				Mask;
+var UIPanel			ChipContainer;
+var array<LW2_UISquadImage> ImageChips; 
+var array<string>		Images;
+var int				InitialSelection; 
+
+delegate OnPreviewDelegate(int iImageIndex);
+delegate OnSetDelegate(int iImageIndex);
+
+simulated function LW2_UIImageSelector InitImageSelector(optional name InitName, 
+															 optional float initX = 500,
+															 optional float initY = 500,
+															 optional float initWidth = 500,
+															 optional float initHeight = 500,
+												 			 optional array<string> initImages,
+												 			 optional delegate<OnPreviewDelegate> initPreviewDelegate,
+															 optional delegate<OnSetDelegate> initSetDelegate,
+															 optional int initSelection = 0)
```

Comment:
Isn't that going a little overboard with indents?

Reply:
I blame robo 😆 I didn't see that, otherwise it would never have got through 👼

Reply:
I blame the author of `UIColorSelector` because this class is almost 1:1 copied from it.

Reply:
My apologies!